tests/ShadowDOM/js/Document.js

```diff
@@ -230,14 +233,18 @@ htmlSuite('Document', function() {
 
     div.offsetHeight;
 
-    var list = document.querySelectorAll('div /deep/ aa');
```

Comment:
@jmesserly Thanks for the feedback, I've just noticed it by looking more closely at the code. Sorry for the noise.